components/atom/switch/src/index.scss

```diff
@@ -1,127 +1,142 @@
 @import '~@schibstedspain/sui-theme/lib/index';
 
-$h-atom-switch--m: $sz-icon-l !default;
-$w-atom-switch--m: 35px !default;
-$m-atom-switch-circle--m: 12px !default;
-$h-atom-switch-container--m: 22px !default;
+/* DEFAULT SIZE */
+$h-atom-switch-container-default: 24px !default;
+$w-atom-switch-container-default: 40px !default;
+$w-atom-switch-circle-default: 22px !default;
+$m-atom-switch-circle-default--active: 16px !default;
+
+/* LARGE SIZE */
+$h-atom-switch-container-large: 32px !default;
+$w-atom-switch-container-large: 56px !default;
+$w-atom-switch-circle-large: 32px !default;
+$m-atom-switch-circle-large--active: 24px !default;
+$h-atom-switch-inputContainer-large: 32px !default;
 
-$h-atom-switch--l: $sz-icon-xl !default;
-$w-atom-switch--l: 51px !default;
-$m-atom-switch-circle--l: 18px !default;
-$h-atom-switch-container--l: 32px !default;
 
 $c-atom-switch--active: color-variation($c-primary, 1) !default;
+$c-atom-switch--active-shadow: rgba(0, 0, 0, .2);
 $c-atom-switch--default: color-variation($c-gray, 3) !default;
 
 .sui-AtomSwitch {
   cursor: pointer;
   display: inline-block;
 
   &-toggleType {
-    .sui-AtomSwitch--container {
+    .sui-AtomSwitch-container {
       margin-top: $p-l;
     }
   }
 
-  &--singleType {
-    .sui-AtomSwitch--container .sui-AtomSwitch--inputContainer {
+  &-singleType {
+    .sui-AtomSwitch-container .sui-AtomSwitch-inputContainer {
       margin-left: $m-l;
     }
   }
 
-  &--container {
+  &-container {
     border: 1px solid transparent;
     display: flex;
-    height: $h-atom-switch--m;
+    height: $h-atom-switch-container-default;
     justify-content: center;
```

Comment:
![image](https://user-images.githubusercontent.com/1083195/43833896-9c06fbe8-9b0c-11e8-85ca-c3c4286356c3.png)
Is it ok that the active zone of the labels is not as height as the switch ?